dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -420,6 +428,18 @@ Simulation::configureFinalStateOutputs() {
   }
 }
 
+void
+Simulation::configureLostEquipmentsOutputs() {
+  // Lost equipments settings
+  if (jobEntry_->getOutputsEntry()->getLostEquipmentsEntry()
+      && jobEntry_->getOutputsEntry()->getLostEquipmentsEntry()->getDumpLostEquipments()) {
+    lostEquipmentsCollection_ = LostEquipmentsCollectionFactory::newInstance();
+
+    setLostEquipmentsExportMode(Simulation::EXPORT_LOSTEQUIPMENTS_XML);
+    setLostEquipmentsOutputFile(createAbsolutePath("lostEquipments.xml", outputsDirectory_));
```

Comment:
Please put this in a  folder similarly to others outputs

Reply:
Done: _lostEquipments.xml_ file in _lostEquipments_ folder